### PR TITLE
Update and add NPC movement scripts - Tirisfal Glades

### PR DESCRIPTION
**Pull request by NickTyrer**
This PR adds some new movement scripts and updates existing ones using waypoints from sniffs where possible.

### CODE REVIEW

**Comment by NickTyrer** [comment]
Need waypoints for creatures in most areas here as non are available in current sniffs.

File: sql/migrations/20200729201638_world.sql [hunk 1]
```diff
@@ -0,0 +1,606 @@
...
+(31933, 46, 2943.459961, 754.460999, 100.375000, 100.000000, 0, 0.000000, 0),
+(31933, 47, 2942.739990, 750.512024, 100.904999, 100.000000, 10000, 3, 0),
+(31933, 48, 2920.169922, 720.804016, 106.163002, 100.000000, 0, 0.000000, 0),
+(31933, 49, 2914.399902, 673.586975, 109.835999, 100.000000, 0, 0.000000, 0),
+(31933, 50, 2915.699951, 630.408997, 100.427002, 100.000000, 0, 0.000000, 0),
+(31933, 51, 2917.889893, 623.309021, 98.392403, 100.000000, 0, 0.000000, 0),
+(31933, 52, 2926.459961, 615.729004, 95.337196, 100.000000, 0, 0.000000, 0);
+
+-- creature 44909 waypoints
+DELETE FROM `creature_movement` WHERE `id` = 44909
```
Comment (alexkulya) [inline sql/migrations/20200729201638_world.sql]: ;

Comment (NickTyrer) [reply sql/migrations/20200729201638_world.sql]: Thanks

**Comment by ratkosrb** [comment]
Really good job!

**Pull request merged**